### ISSUE TITLE
Animation freezing

### ISSUE THREAD

**Issue by seopromo** [metafizzy/isotope#1373]
Hello! 
I often meet issue with animation like these http://prntscr.com/if0e4b 
Animation just freezing and don't display correctly
For example on the page http://atiframe.secretlab.pw/digital-marketing/
It works not always. I saw it many times in Firefox, but my teammate also saw the issue in chrome.
I did try animationOptions:   duration: 0 and 500. It doesn't matter.
Do you have any idea?
Best regards

**Comment by desandro** [comment]
I'm sorry to see you're having trouble with Isotope. Could you provide a **reduced test case**? See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/isotope/blob/master/.github/contributing.md#submitting-issues).

Isotope does not support `animationOptions`. That was a v1 feature. Isotope is current v3. Are you using the latest version? https://isotope.metafizzy.co

**Comment by seopromo** [comment]
The version is v3.0.5
http://atiframe.secretlab.pw/wp-content/plugins/secretlab_shortcodes/js/isotope.pkgd.min.js

I have just pages with the issue. I did research it. The issue doesnt appear on all pages. It does look like the gallery took animation option from another js element on the page. I made a video how it works https://youtu.be/VtOrHg3I_Nk

Im still dont understand how does the bug work.

Can i disable animation? I would like the gallery load without any animation.
